src/Pages/BookPage.js

```diff
@@ -1,153 +1,210 @@
+    image = (book.get('image')) ? book.get('image') : [{ url: Logo }];
+    readAloudURL = (book.get('read_aloud_link')) ? book.get('read_aloud_link') : null;
+    bookshopURL = (book.get('bookshop_link')) ? book.get('bookshop_link') : null;
+    educatorURLs = (book.get('educator_guide_links')) ? book.get('educator_guide_links').split('\n') : [];
+    identityTags = (book.get('identity_tags')) ? book.get('identity_tags') : [];
+    ageMin = (book.get('age_min')) ? book.get('age_min') : -1;
+    ageMax = (book.get('age_max')) ? book.get('age_max') : -1;
+    gradeMin = (book.get('grade_min')) ? book.get('grade_min') : -1;
+    gradeMax = (book.get('grade_max')) ? book.get('grade_max') : -1;
+  }
+
+  if (author) {
+    authorName = (author.get('name')) ? author.get('name') : authorName;
+    authorID = (author.get('id')) ? author.get('id') : authorID;
+  }
+
+  if (illustrator) {
+    illustratorName = (illustrator.get('name')) ? illustrator.get('name') : illustratorName;
+    illustratorID = (illustrator.get('id')) ? illustrator.get('id') : illustratorID;
+  }
+
+  const imageURL = image[0].url;
+
+  const synopsisProps = {
+    title,
+    authorName,
+    authorID,
+    illustratorName,
+    illustratorID,
+    desc,
+    imageURL,
+    bookshopURL,
+    readAloudURL,
+    identityTags,
+    ageMin,
+    ageMax,
+    gradeMin,
+    gradeMax,
+  };
+
+  const isValidUrl = (string) => {
+    /* Validate url given by TCK. URL must start with http or https protocol. */
+    let url;
+
+    try {
+      url = new URL(string);
+    } catch (_) {
+      return false;
+    }
+    return url.protocol === 'http:' || url.protocol === 'https:';
+  };
+
+  const educatorLinks = educatorURLs.map((url) => (
+    <ListItem button component="a" href={url} key={url} alignItems="center">
+      <ListItemText sx={{ textAlign: 'center' }} primary={url} />
+    </ListItem>
+  ));
+
+  educatorLinks.filter((string) => isValidUrl(string));
+
+  if (title === 'Untitled Book') {
+    return (<h1>{'Sorry, we couldn\'t retrieve this book from our library 😔'}</h1>);
+  }
+
+  return (
+    <Paper variant="outlined">
+      <CollectionsCarousel
+        elementArray={collections}
+        slidesAtATime={6}
+        prevArrow={LeftArrow}
+        nextArrow={RightArrow}
+        widthPercent={100}
+        spaceBetweenEntries={16}
+        swiperHeight={120}
+        cardImageHeightPercent={80}
+        cardImageWidthPercent={80}
+      />
+      <BookSynopsis {...synopsisProps} />
+      {(readAloudURL) ? (
+        <Box sx={styles.iframeContainer}>
+          <iframe
+            style={styles.iframe}
+            src={`https://www.youtube.com/embed/${readAloudURL.split('watch?v=')[1]}`}
+            frameBorder="0"
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+            allowFullScreen
+            title="Embedded youtube"
+          />
+        </Box>
+      ) : <div />}
```

Comment:
Rather than using the tertiary logic statement you can just use an && and not display anything if there isn't a url! is the div important for the styling?
```suggestion
      {(readAloudURL) && (
        <Box sx={styles.iframeContainer}>
          <iframe
            style={styles.iframe}
            src={`https://www.youtube.com/embed/${readAloudURL.split('watch?v=')[1]}`}
            frameBorder="0"
            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
            allowFullScreen
            title="Embedded youtube"
          />
        </Box>
      )}
```